### ISSUE TITLE
Mel_Basis kernel 

### ISSUE THREAD

**Issue by gunikavashisht13472** [KinWaiCheuk/nnAudio#114]
Hello! While training my model, I used MelSpectrogram function and realized that after few epochs(15-10), the mel basis(mel filterbank) are stuck and donot change at all. I even tried increasing the learning rate and loss regularization. Could you let me know what is wrong with it?

**Comment by KinWaiCheuk** [comment]
The mel filterbanks should update according to the gradient. Can you compare the gradient for the mel filterbanks between the first few epochs and the epochs when it stops updating?
I feel like that the gradient becomes zeros and hence it stops updating.
